README.md

```diff
@@ -40,44 +41,31 @@ During development, the SCSS files are linted on every `commit` and built on eve
 
 Browser-specific properties are generated at build-time through the [PostCSS autoprefixer](https://github.com/postcss/autoprefixer).
 
-### Working with the themes monorepo
+### Working with This Monorepo
 
-The repository uses [lerna](https://github.com/lerna/lerna/) to ship the multiple theme packages from a single git repository. For details why this is a good idea for the themes, see [this issue](https://github.com/telerik/kendo-theme-default/issues/720).
+The repository uses [lerna](https://github.com/lerna/lerna/) to ship the multiple theme packages from a single Git repository. For details why this is a good idea for the themes, see [this issue](https://github.com/telerik/kendo-theme-default/issues/720).
 
-To set up the repository and cross-link the theme packages, run the following commands:
+To set up the monorepo:
 
-```bash
-# install root-level dependencies
-npm install
+1. Run `npm install` to install the root-level dependencies.
+1. Run `npx lerna bootstrap` to install the theme dependencies and cross-link the theme packages.
 
-# installs theme dependencies and cross-links theme packages
-npx lerna bootstrap
-```
+  > The `npx` command runs packages from `./node_modules/.bin` without requiring a global install.
 
-> `npx` runs packages from `./node_modules/.bin`, without requiring a global install
+1. After running these commands and in the root of the repository:
+  1. Run `npx lerna run lint` to run the linting over all the themes.
+  1. Run `npx lerna run build` to build all the themes.
+  1. Run `npm test` to run the tests (lint, JS tests, and build).
```

Comment:
These commands are not part of the setup, but a list of helpful commands that can be run in the root of the repo. Perhaps this should be outside the setup instructions.